docs/packaging-applications/build-servers/tfs-azure-devops/using-octopus-extension/extension-compatibility.md

```diff
@@ -5,22 +5,24 @@ description: This guide explains the Octopus extension versions that are compati
 
 ## Octopus Extension Versions
 
-There are three distinct versions (or version ranges) you need to worry about with respect to the Octopus Extension:
+There have been four major versions of the Octopus Extension:
 
-- [**Version 1.2.x**](https://s3-eu-west-1.amazonaws.com/octopus-downloads/tfs-2015-extension/octopusdeploy.octopus-deploy-build-release-tasks-1.2.28.vsix) - now obsolete, but still usable for older versions of TFS and Azure DevOps
-- [**Version 2.0.199**](https://s3-eu-west-1.amazonaws.com/octopus-downloads/tfs-2015-extension/octopusdeploy.octopus-deploy-build-release-tasks-2.0.199.vsix) - a specific "version 2" build for TFS 2015 Update 2, TFS 2015 Update 3, TFS 2015 Update 4, and TFS 2017 RTM
-- [**Version 3.0.x**](https://marketplace.visualstudio.com/items?itemName=octopusdeploy.octopus-deploy-build-release-tasks) - the current, most recent version of the extension, for Azure DevOps and TFS 2017 Update 1 and above
+- [**Version 1.2.x**](https://s3-eu-west-1.amazonaws.com/octopus-downloads/tfs-2015-extension/octopusdeploy.octopus-deploy-build-release-tasks-1.2.28.vsix) - obsolete, but still usable for older versions of TFS and Azure DevOps
+- [**Version 2.0.199**](https://s3-eu-west-1.amazonaws.com/octopus-downloads/tfs-2015-extension/octopusdeploy.octopus-deploy-build-release-tasks-2.0.199.vsix) - for TFS 2015 Update 2, TFS 2015 Update 3, TFS 2015 Update 4, and TFS 2017 RTM
+- [**Version 3.0.x**](https://octopus-downloads.s3-eu-west-1.amazonaws.com/tfs-2015-extension/octopusdeploy.octopus-deploy-build-release-tasks-3.0.222.vsix) - download for TFS 2017 Update 1, or use v3 tasks in the current extension for Octopus Server versions prior to 2019.1
```

Comment:
```suggestion
- [**Version 3.0.x**](https://octopus-downloads.s3-eu-west-1.amazonaws.com/tfs-2015-extension/octopusdeploy.octopus-deploy-build-release-tasks-3.0.222.vsix) - download for TFS 2017 Update 1
```

Since TFS2017 update 1 can't install the most recent version of the current extension they wouldn't be able to use the `v3` tasks, so we should remove this statement.

Reply:
I meant that bit to refer to users using a pre-2019.1 version of Octopus, with TFS 2017 Update 2 or newer. But is that too obscure to bother documenting?

Reply:
I don't think that's necessary to document, hopefully the descriptions in the tasks themselves would make that clear

Reply:
👍